drools-benchmarks-parent/drools-benchmarks-reliability/src/main/java/org/drools/benchmarks/reliability/AbstractReliabilityBenchmark.java

```diff
@@ -74,25 +88,33 @@ public String getInfinispanStorageMode() {
     public void setupEnvironment() {
         FileUtils.deleteDirectory(Path.of(GLOBAL_STATE_DIR));
 
-        if (mode != NONE) {
-            System.setProperty(INFINISPAN_STORAGE_MODE, mode.getInfinispanStorageMode());
-        }
+        if (module == Module.H2MVSTORE){
+            H2MVStoreStorageManager.cleanUpDatabase();
+            System.setProperty(DROOLS_RELIABILITY_MODULE_TEST, "H2MVSTORE");
+            configureServicePriorities();
+        }else { // assuming infinispan is the default module
+            System.setProperty(DROOLS_RELIABILITY_MODULE_TEST, "INFINISPAN");
+            configureServicePriorities();
```

Comment:
Please move this `configureServicePriorities();` to line-110.   `configureServicePriorities() triggers storage initialization, so it should be called after all system properties are set.

Reply:
Ok, done.